<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出售商品</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_all" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>670</width>
     <height>340</height>
    </rect>
   </property>
   <widget class="SaleForm" name="widget_sale" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>650</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="FindForm" name="widget_find" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>650</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="ChangePasswdForm" name="widget_change_passwd" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>650</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="AddGoodsForm" name="widget_add_goods" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>8</y>
      <width>650</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="StorageForm" name="widget_storage_goods" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>650</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>651</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_sale_goods">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>出售商品</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn_storage_goods">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品入库</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="btn_add_goods">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>添加商品</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="btn_find_goods">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品查询</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="8">
     <widget class="QPushButton" name="btn_change_passwd">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SaleForm</class>
   <extends>QWidget</extends>
   <header>saleform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FindForm</class>
   <extends>QWidget</extends>
   <header>findform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChangePasswdForm</class>
   <extends>QWidget</extends>
   <header>changepasswdform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AddGoodsForm</class>
   <extends>QWidget</extends>
   <header>addgoodsform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StorageForm</class>
   <extends>QWidget</extends>
   <header>storageform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
